torchao/quantization/README.md

```diff
@@ -202,6 +202,17 @@ We also have a unified quantized tensor subclass that implements how to get a qu
 #### Layouts
 We extended the `layout` concept to represent different packing formats for a tensor. `AffineQuantizedTensor` supports `plain` and `tensor_core_tiled` layout. `plain` layout is used for `int8_weight_only` and `int8_dynamic_activation_int8_weight` and also as a default layout. `tensor_core_tiled` layout is used for `int4_weight_only` quantization and is packing the weights in a format that is compatible with tinygemm [int4mm](https://github.com/pytorch/pytorch/blob/39357ba06f48cda7d293a4995aa5eba2a46598b5/aten/src/ATen/native/native_functions.yaml#L4138) kernels.
 
+### Zero Point Domains
+```ZeroPointDomain``` is used to control the data types of zero points. ```None``` represents symmetric quantization, while ```ZeroPointDomain.FLOAT``` and ```ZeroPointDomain.INT``` indicate asymmetric quantization. For detailed implementation of different zero point data types, refer to [the reference implementation]((../../test/quantization/test_quant_primitives.py)).
```

Comment:
we have MappingType to mean symmetric or asymmetric quant, zero_point_domain is referring to zero_point, None means zero_point is None, FLOAT means zero_point is in the floating point domain and INT means integer domain, please see: https://github.com/pytorch/ao/blob/46b8796412eb350d1923091892850582d32737d0/torchao/quantization/quant_primitives.py#L68

Reply:
https://github.com/pytorch/ao/pull/1401/commits/e4237e006d11624633555ac40283ee6dd8349f34